src/backend/distributed/master/citus_create_restore_point.c

```diff
@@ -171,28 +174,35 @@ CreateRemoteRestorePoints(char *restoreName, List *connectionList)
 	{
 		MultiConnection *connection = (MultiConnection *) lfirst(connectionCell);
 
-		int querySent = SendRemoteCommandParams(connection, CREATE_RESTORE_POINT_COMMAND,
-												parameterCount, parameterTypes,
-												parameterValues);
-		if (querySent == 0)
+		if (GroupForNode(connection->hostname, connection->port) != 0)
```

Comment:
Probably not necessary, since `ActivePrimaryNodeList()` doesn't include the coordinator. Maybe just rename `OpenConnectionsToAllNodes()` to `OpenConnectionsToAllWorkerNodes()` and add some test.